### ISSUE TITLE
Entity wizard is broken

### ISSUE THREAD

**Issue by LordFeathers** [JannisX11/blockbench-plugins#440]
### What are you trying to do, and what do you expect to happen?

I am trying to use the entity wizard to make a custom mob, but when loaded into Minecraft, it just shows the texture and animation of the default mob I chose to work off of.

### What happens instead?

When looking into the file, it doesn't even save the new model into it, when I change it to the actual one, in Minecraft it says to check the error log, this pack can't run. (I exported it as a .McAddon file).

### Model format in which the issue occurs

Geometry.json

### Blockbench variant

Program

### Blockbench version

4.8.3

### Operating System

Windows 11

### Installed Blockbench plugins

Entity wizard

**Comment by JannisX11** [comment]
Are you by any chance exporting and loading the mcaddon file only before modifying the model? In that case it would make sense why it doesn't display your model.
I would recommend exporting directly to the folder instead.

**Comment by LordFeathers** [comment]
No, I re-exported as a.mcaddon after animating, modeling, and texturing. When I export directly to development behavior and resource pack folders in com.Mojang, it does work though. But for some reason when I zip the files and .mcaddonify it, Minecraft says failed to import pack, check error log for details. But that part is probably on my end.
